src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from "react"
 import ReactDOM from "react-dom"
-import App from "./App"
+import AppAuth from "./AppAuth"
 import * as serviceWorker from "./serviceWorker"
+import { AuthProvider } from "./auth-context"
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <AuthProvider>
```

Comment:
Let's wrap our App not in this file but in `src/App.js` you can place it under the Router component so that it wraps the whole App. There is not right or wrong reason for this changes but it is preferable to keep this file as clean as possible.